sdk/evaluation/azure-ai-evaluation/samples/evaluation_samples_threshold.py

```diff
@@ -61,6 +61,12 @@ def evaluation_classes_methods_with_thresholds(self):
                     },
                 },
             },
+            tags={
```

Comment:
Is this change required ? This sample if for thresholds. Please move the tags related changes to sample for tags.